sphinx/source/docs/user_guide/server.rst

```diff
@@ -762,38 +760,45 @@ thread, e.g. when using ``bokeh.client`` to create sessions from a web
 framework such as Django or Flask. In these cases calling
 ``loop_until_closed()`` from another thread may be an option. Another
 possibility is to not use ``bokeh.client`` at all. See the section
-`userguide_server_embedding`_ for some alternatives.
+:ref:`userguide_server_embedding` for some alternatives.
 
 .. _userguide_server_deployment:
 
 Deployment Scenarios
 --------------------
 
-With an application, we can run it just locally any time we want to interact
-with it. Or we can share it with other people, and they can run it locally
-themselves in the same manner. But we might also want to deploy the application
-in a way that other people can access it. This section describes some of the
-considerations that arise in that case.
+With an application we are developing, we can run it locally any time we want to interact
+with it. To share it with other people who are able to install the required
+python stack, we can share the application with them, and let them run it locally
+themselves in the same manner. However, we might also want to deploy the application
+in a way that other people can access it as a service:
+* without having to install all of the prerequisites
+* without needing to have the source code
+* like any other webpage
+
+This section describes some of the considerations that arise when deploying
+Bokeh server applications as a service for others to use.
```

Comment:
As a general rule, does the Bokeh project mention Continuum's Enterprise offerings where applicable?  I'm asking because this section could possibly use a pointer to [the magic button](https://youtu.be/ZJ8dTg_eKzc) from Anaconda Enterprise v5.0 later this year, right @koverholt ?  However, if we don't point to awesome Enterprise things from the Bokeh docs, I'll remember that as a rule.